src/diffpy/srmise/tests/test_dataclusters.py

```diff
@@ -0,0 +1,140 @@
+import numpy as np
+import pytest
+
+from diffpy.srmise.dataclusters import DataClusters
+
+
+@pytest.mark.parametrize(
+    "inputs, expected",
+    [
+        (
+            {
+                "input_x": np.array([1, 2, 3]),
+                "input_y": np.array([3, 2, 1]),
+                "input_res": 4,
+            },
+            {
+                "x": np.array([]),
+                "y": np.array([]),
+                "data_order": np.array([], dtype=np.int32),
+                "clusters": np.array([[]], dtype=np.int32),
+                "res": 0,
+                "current_idx": 0,
+                "lastcluster_idx": None,
+                "lastpoint_idx": None,
+                "status": 0,
+            },
+        ),
+        (
```

Comment:
what behavior is this testing that is different than the previous case?

Reply:
Here I'm testing clearing object with multiple values in x & y lists, but since it is actually a method to reset everything into the very beginning state, I'm not sure if this case is necessary.

Reply:
correct.  in both cases you are testing arrays.  That they are different lengths is kind of irrelevant.

I think for the clear function, we don't need to test too many different cases, if any.  For other functions we may want to test more different cases.